components/espcoredump/src/core_dump_flash.c

```diff
@@ -82,6 +86,17 @@ void esp_core_dump_flash_init(void)
     s_core_flash_config.partition.start      = core_part->address;
     s_core_flash_config.partition.size       = core_part->size;
     s_core_flash_config.partition.encrypted  = core_part->encrypted;
+
+#if CONFIG_ESP_COREDUMP_FLASH_NO_OVERWRITE
+    uint32_t core_size;
+    esp_err_t err = esp_partition_read(core_part, 0, &core_size, sizeof(core_size));
+    if (err == ESP_OK && core_size == BLANK_COREDUMP_SIZE) {
```

Comment:
If the read partition caused an error it may be better to print an error:
```
if (err == ESP_OK) {
    s_core_flash_config.partition.empty = (core_size == BLANK_COREDUMP_SIZE);
} else {
    ESP_COREDUMP_LOGE("Could not read coredump partition size (%d)", err);
}
```

Reply:
I've done this and reused the same message as elsewhere in the file.